_docs/howto/deploy/change-gcp-deployment.md

```diff
@@ -1,28 +1,64 @@
 ---
-title: Changing a Deployment
+title: Updating Forseti
 order: 104
 ---
 #  {{ page.title }}
 
-This page describes how to change a Forseti Security deployment. Most changes
-to a deployment are usually to the deployment properties, such as the
-instance type or `src-path` when you want your deployment to run a certain 
-version of Forseti Security. To update your deployment, follow the process below:
+### Update local installation
+If you want to update your local version of Forseti, pull in the latest changes from
+git:
 
-  1. Edit `deploy-forseti.yaml` and update the values you want to change.
+```bash
+$ cd forseti-security
+$ git pull
+```
+
+And then run the python installation:
+
+```bash
+$ python setup.py install
+```
+
+### Change deployment properties
+If you need to change any of the properties for your Forseti deployment, such as the
+`release-version`, follow the process below:
+
+  1. If you **do not** need to update your Forseti version, continue on to step 3.
```

Comment:
This could get confusing in future edits. It might be best to use an include for the common steps and have two separate sections - one in which I update Forseti version, and another where I don't.

That said - how do I know if I need to update my Forseti version?

Reply:
The user will find out about Forseti updates by coming to the Github website. We don't push users to get the newest release version.

Reply:
Gotcha, okay, so what I'm getting here is that updating the local installation is optional. If you do that, then you need to do steps 1-2 to update Forseti version. Then you edit deploy-forseti.yaml to make the other changes you want. What I'd suggest is to make separate headings for those, along these lines:

```
## Update local installation (optional)
  (list of steps)

### Update Forseti version
After you update your local installation, follow the steps below to update your Forseti version:
  (list of steps)
Next, continue to change your deployment properties.

## Change deployment properties
To update your deployment, follow the steps below:

1. Edit `deploy-forseti.yaml` ...
```
Let me know if you'd prefer me to submit an edit to the file instead of trying to make sense of my comments :)

Reply:
Sorry wasn't clearer -- updating local installation isn't a step; it's for people who want to update their local installation.

To update the version for a Deployment Manager deployment, you should:

1. Pull down the latest code and checkout the appropriate version (git command)
2. Update necessary deployment template (deploy-forseti.yaml), including the Forseti version
3. Run the deployment manager update (gcloud command)


This is basically the same thing as changing a deployment. The only explicit instruction is telling the user to update the Forseti version property.